docs/Design-Areas/LZA-ASA-Security.md

```diff
@@ -11,20 +11,22 @@ Azure Spring Apps is the new name for the Azure Spring Cloud service. Security c
 A security control is a quality or feature of an Azure service that contributes to the service's ability to prevent, detect, and respond to security vulnerabilities. For each control, we use Yes or No to indicate whether it is currently in place for the service. We use N/A for a control that is not applicable to the service.
 
 ### Azure Spring Apps Landing Zone - Topology
-Public:
-
-<img width="1042" alt="ASA-LZA-Public-screenshot" src="https://github.com/Azure/azure-spring-apps-reference-architecture/blob/main/images/architecture-public.svg">
-
-Private:
-
-<img width="1042" alt="ASA-LZA-Private-screenshot" src="https://github.com/Azure/azure-spring-apps-reference-architecture/blob/main/images/architecture-private.svg">
 
+![Architectural diagram for the secure baseline scenario.](https://github.com/Azure/azure-spring-apps-reference-architecture/blob/aarthiem-patch-1/Scenarios/ASA-Secure-Baseline/media/asa-eslz-securebaseline.jpg)
 
 ### Azure Spring Apps Landing Zone - Azure Components
-Pending 
-| Component | Version | Location |
-|-------------|---------------|---------------|
 
+| Component | Overview | Location |
+|-------------|---------------|---------------|
+| Azure Firewall |<p>Azure Firewall is a cloud-native and intelligent network firewall security service that provides the best of breed threat protection for your cloud workloads running in Azure. It's a fully stateful, firewall as a service with built-in high availability and unrestricted cloud scalability. It provides both east-west and north-south traffic inspection. To learn what's east-west and north-south traffic, see <a href="/en-us/azure/architecture/framework/security/design-network-flow#east-west-and-north-south-traffic" data-linktype="absolute-path">East-west and north-south traffic</a>. Azure Firewall is offered in three SKUs: Standard, Premium, and Basic.</p>|Hub|
```

Comment:
What is the root of this link? It refers to /en-us without an FQDN.

Reply:
Hey, I resolved the issues for the URL...changed it to:  Scenarios/ASA-Secure-Baseline/media/asa-eslz-securebaseline.jpg